src/pages/billing/Project.js

```diff
@@ -19,20 +20,76 @@ import * as Utils from 'src/libs/utils'
+const workspaceLastModifiedWidth = 150
+
+const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ sort, onSort }) => {
+  return div({ role: 'row', style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem', marginBottom: '0.5rem' } }, [
+    div({ role: 'columnheader', 'aria-sort': ariaSort(sort, 'name'), style: { flex: 1 } }, [
+      h(HeaderRenderer, { sort, onSort, name: 'name' })
+    ]),
+    div({ role: 'columnheader', 'aria-sort': ariaSort(sort, 'createdBy'), style: { flex: 1 } }, [
+      h(HeaderRenderer, { sort, onSort, name: 'createdBy' })
+    ]),
+    div({ role: 'columnheader', 'aria-sort': ariaSort(sort, 'lastModified'), style: { flex: `0 0 ${workspaceLastModifiedWidth}px` } }, [
+      h(HeaderRenderer, { sort, onSort, name: 'lastModified' })
+    ])
+  ])
+})
+
+const rowBase = { display: 'flex', alignItems: 'center', width: '100%' }
+
+const ExpandedInfoRow = Utils.memoWithName('ExpandedInfoRow', ({ title, details, additionalInfo }) => {
+  const expandedInfoStyles = {
+    row: { ...rowBase, marginTop: '0.5rem' },
+    title: { fontWeight: 600, width: '20%', paddingRight: '1rem' },
+    details: { flexGrow: 1, width: '20%', paddingRight: '1rem', ...Style.noWrapEllipsis },
+    additionalInfo: { flexGrow: 1 }
   }
-}
 
-const WorkspaceCard = Utils.memoWithName('WorkspaceCard', ({ workspace }) => div({
-  style: Style.cardList.longCardShadowless
-}, [
-  div({ style: styles.workspaceCardField }, [workspace.name]),
-  div({ style: styles.workspaceCardField }, [workspace.createdBy]),
-  div({ style: { flex: `0 0 ${workspaceLastModifiedWidth}px` } }, [workspace.lastModified])
-]))
+  return div({ style: expandedInfoStyles.row, role: 'row' }, [
+    div({ role: 'cell', style: expandedInfoStyles.title }, [title]),
+    div({ role: 'cell', style: expandedInfoStyles.details }, [details]),
+    div({ role: 'cell', style: expandedInfoStyles.additionalInfo }, [additionalInfo])
+  ])
+})
+
+const WorkspaceCard = Utils.memoWithName('WorkspaceCard', ({ workspace, isExpanded, onExpand }) => {
+  const { namespace, name, createdBy, lastModified, googleProject, billingAccountName } = workspace
+  const workspaceExpandIconSize = 20
+  const workspaceCardStyles = {
+    field: {
+      ...Style.noWrapEllipsis, flex: 1, width: `calc(50% - ${workspaceLastModifiedWidth / 2}px)`, paddingRight: '1rem'
+    },
+    row: rowBase,
+    expandedInfoContainer: { display: 'flex', flexDirection: 'column', width: '100%' }
+  }
+
+  return div({ role: 'row', style: { ...Style.cardList.longCardShadowless, flexDirection: 'column' } }, [
+    div({ style: workspaceCardStyles.row }, [
+      div({ role: 'rowheader', style: { ...workspaceCardStyles.field, display: 'flex', alignItems: 'center' } }, [
+        h(Link, {
+          style: { fontWeight: 600, fontSize: 16, ...Style.noWrapEllipsis },
+          href: Nav.getLink('workspace-dashboard', { namespace, name })
+        }, [name]),
+        h(Link, {
```

Comment:
Does this need anything else for accessibility @andrewmarcus? Or do our components handle all that for us?

Reply:
Well, you'll need the link to have either a label or a tooltip which becomes a label once #2525 is merged. The `aria-expanded` is perfect though